Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -297,6 +297,15 @@ public struct TestRequirement: Decodable {
         )
     ]
 
+    public static let changeStreamOnCollectionSupport = TestRequirement(
```

Comment:
this and the various usages of it are not exactly related to the Evergreen work, however as I was looking through the logs to make sure tests ran correctly I noticed we were incorrectly skipping a bunch of tests which we had hardcoded the supported topologies for and I forgot to add `.loadBalanced` to so I updated those as well so they would actually run.